src/test/java/uk/gov/pay/api/utils/mocks/CreateChargeRequestParams.java

```diff
@@ -0,0 +1,89 @@
+package uk.gov.pay.api.utils.mocks;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+public class CreateChargeRequestParams {
+    
+    private final int amount;
+    private final String returnUrl, description, reference;
+    private final Map<String, String> metadata;
```

Comment:
I think this needs to be `Map<String, Object>` so we can include all of `String`, `Number` and `Boolean`.